docs/en/usage/recipes/bundler/sidecar.md

```diff
@@ -28,7 +28,7 @@ Here is a sample to illustrate the configuration, this is not a complete `tauri.
 This way, you may [execute commands with Rust](https://doc.rust-lang.org/std/process/struct.Command.html) in your Tauri application.
 
 <Alert title="Note">
-Tauri provides some utilitary functions to handle standard cases (like loading platform specific binaries), such as:
+Tauri provides some utilitarian functions to handle standard cases (like loading platform specific binaries), such as:
```

Comment:
I think we should just drop `utilitarian` here, it sounds a little off to me.

Reply:
Ya I agree.